src/jwt/rsa-verifier.js

```diff
@@ -0,0 +1,76 @@
+/*
+Based on the work of Tom Wu
+http://www-cs-students.stanford.edu/~tjw/jsbn/
+http://www-cs-students.stanford.edu/~tjw/jsbn/LICENSE
+*/
+
+import {BigInteger} from 'jsbn';
+import SHA256 from 'crypto-js/sha256';
+
+var DigestInfoHead = {
+  sha1: '3021300906052b0e03021a05000414',
+  sha224: '302d300d06096086480165030402040500041c',
+  sha256: '3031300d060960864801650304020105000420',
+  sha384: '3041300d060960864801650304020205000430',
+  sha512: '3051300d060960864801650304020305000440',
+  md2: '3020300c06082a864886f70d020205000410',
+  md5: '3020300c06082a864886f70d020505000410',
+  ripemd160: '3021300906052b2403020105000414',
+};
+
+var DigestAlgs = {
+  sha256: SHA256,
+};
+
+function RSAVerifier(modulus, exp) {
+  this.n = null;
+  this.e = 0;
+
+  if (modulus != null && exp != null && modulus.length > 0 && exp.length > 0) {
```

Comment:
Shouldn't this be checking for truthy values instead of `!= null`? Like `if (modulus && exp && ...)`

Reply:
I'm not 100% sure. We will need to check.

Reply:
Correct. Changed.